scipy/stats/tests/test_stats.py

```diff
@@ -3997,7 +3997,7 @@ def test_ttest_ind_randperm_alternative2(self):
         N = 50
         a = np.random.rand(N, 4)
         b = np.random.rand(N, 4)
-        options_p = {'permutations': 500000, "random_state":0}
+        options_p = {'permutations': 50000, "random_state":0}
```

Comment:
Let's remove this change, as it's not related to this PR. I'm addressing it in gh-13527. Interesting to know that 50000 is enough, though.

Reply:
I had CI failing because this seemingly  unrelated test was failing https://github.com/scipy/scipy/pull/13298/commits/ab15280613e356e83a3b9869d0cb09db3b5f6c61. See permalink in commit message for details. Also was failing on my WSL (more than 2 gb memory required apparently). no problem reverting but CI will fail again i presume.

Btw 50k get close but not as close, had to change tolerance to 0.01.

Reply:
Sounds good. When gh-13527 merges it will pass again.

Reply:
BTW gh-13527 merged.